Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04 AS build
+
+WORKDIR /build
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y build-essential libboost-dev libboost-date-time-dev libmysql++-dev libglib2.0-dev libgsl-dev
+COPY . /build
+RUN make
+
+
+FROM ubuntu:18.04
+
+WORKDIR /srv
+RUN useradd -ms /bin/bash aga
+ENV PATH="/srv:${PATH}"
```

Comment:
I think we should reorder the RUN and ENV. We also don't need to equal sign as we only have one variable here.

Reply:
Oh, that ENV doesn't even need to be here any more, should be removed. Will fix. (see the modification of /etc/environment below, which doesn't clobber the path with the builder's path)

Reply:
When this is updated ill merge the PR.

Reply:
Turns out that `ENV` was load-bearing after all. Guess I'll just have to live with it. Update coming soon.